docker/service/src/main/java/uk/nhs/adaptors/scr/utils/FhirHelper.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.adaptors.scr.utils;
+
+import org.hl7.fhir.r4.model.Bundle;
+
+public class FhirHelper {
+    private Bundle bundle;
+
+    public FhirHelper(Bundle bundle) {
+        this.bundle = bundle;
+    }
+
+    public <T> T getDomainResource(Class<T> resourceType) {
```

Comment:
Might be better to avoid nulls and use functional approach:
```
public <T extends BaseResource> Optional<T> getDomainResource(Class<T> resourceType) {
    return this.bundle.getEntry().stream()
        .map(BundleEntry::GetResource)
        .filter(resource -> resource.getClass() == resourceType)
        .findFirst();
        // what iff there are more than 1 resource of a given type?
        // you could use: 
        .reduce((a, b) -> { throw new IllegalStateException("There is more than 1 resource of type ...")})
        .findFirst();
}
```
and even make this a static function that accepts both `Bundle` and `Class<T>` as parameters